Misc/NEWS.d/next/Core and Builtins/2019-04-16-11-52-21.bpo-27987.n2_DcQ.rst

```diff
@@ -0,0 +1,3 @@
+pymalloc returns memory blocks aligned by 16 bytes instead of 8 bytes on 64
+bit platforms, as recent compilers assume malloc returns 16 byte aligned
+memory block. Patch by Inada Naoki.
```

Comment:
"as recent compilers assume malloc returns 16 byte aligned memory block" that's not the root issue. The root issue is that Python doesn't conform to the x86-64 ABI.

Reply:
How about "... to confirm x86-64 ABI.  Recent compilers assume this alignment more often."

Reply:
Yes, but it's "conform" not "confirm" :-)

```
pymalloc returns memory blocks aligned by 16 bytes, instead of 8 bytes, on
64-bit platforms to conform x86-64 ABI. Recent compilers assume this alignment
more often. Patch by Inada Naoki.
```